RoadMap_VK/RoadMap_VK/SignInViewController.swift

```diff
@@ -0,0 +1,140 @@
+// ViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран входа в приложение
+final class SignInViewController: UIViewController {
+    // MARK: - Constants
+    private enum Constants {
+        static let segueIdentifier = "signInSegue"
+        
+        enum AdminProfile {
+            static let username = "admin"
+            static let password = "123"
+        }
+        
+        enum SignInResult {
+            static let success = "Success"
+            static let fail = "Fail"
+        }
+        
+        enum AlertText {
+            static let errorTitle = "Ошибка!"
+            static let errorText = "Неправильный логин или пароль"
+            static let okActionText = "Ok"
+        }
+    }
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var scrollView: UIScrollView!
```

Comment:
```suggestion
    @IBOutlet private weak var scrollView: UIScrollView!
```
IBOutlet private исправь остальные